haskell-language-server.cabal

```diff
@@ -205,69 +205,59 @@ executable haskell-language-server-wrapper
-                      FormatSpec
-                     -- , FunctionalBadProjectSpec
-                     -- , FunctionalCodeActionsSpec
-                     -- , FunctionalLiquidSpec
-                     , FunctionalSpec
-                     -- , HaReSpec
-                     -- , HieBiosSpec
-                     -- , HighlightSpec
-                     -- , HoverSpec
-                     , PluginSpec
-                     -- , ProgressSpec
-                     -- , ReferencesSpec
-                     -- , RenameSpec
-                     -- , SymbolsSpec
-                     -- , TypeDefinitionSpec
-                     , Utils
-                     , Paths_haskell_language_server
-
-  hs-source-dirs:
-      test/functional
-  ghc-options:
-         -Wall
-         -Wredundant-constraints
-         -Wno-name-shadowing
-         -threaded -rtsopts -with-rtsopts=-N
+  import:               agpl
+  type:                 exitcode-stdio-1.0
+  default-language:     Haskell2010
+  build-tool-depends:   haskell-language-server:haskell-language-server
+                      , cabal-helper:cabal-helper-main
+                      , ghcide:ghcide-test-preprocessor
+  build-depends:        base >=4.7 && <5
+                      , aeson
+                      , data-default
+                      , directory
+                      , filepath
+                      , haskell-language-server
+                      , haskell-lsp
+                      , haskell-lsp-types
+                      , hls-test-utils
+                      , hspec-expectations
+                      , lens
+                      , lsp-test >= 0.10.0.0
+                      , tasty
+                      , tasty-hunit
+                      , tasty-rerun
+                      , text
+                      , unordered-containers
+  hs-source-dirs:       test/functional
+  main-is:              Main.hs
+  other-modules:        Command
+                      , Completion
+                      , Deferred
+                      , Definition
+                      , Diagnostic
+                      , Format
+                      , FunctionalBadProject
+                      , FunctionalCodeAction
+                      , FunctionalLiquid
+                      , HieBios
+                      , Highlight
+                      , Progress
+                      , Reference
+                      , Rename
+                      , Symbol
+                      , TypeDefinition
+  ghc-options:          -Wall
+                        -Wno-name-shadowing
+                        -threaded -rtsopts -with-rtsopts=-N
   if flag(pedantic)
-     ghc-options:      -Werror
-  main-is: Main.hs
-  -- other-modules:
-  --       Development.IDE.Test
-  --       Development.IDE.Test.Runfiles
+     ghc-options: -Werror -Wredundant-constraints
 
 library hls-test-utils
   import: agpl
   hs-source-dirs:      test/utils
-  exposed-modules:     TestUtils
+  exposed-modules:     Test.HIE.Util
```

Comment:
This should be `Test.Hls.Util`? Or `Test.HLS.Util`, but personally, I like the former more

Reply:
added to next push